packages/api-reference/src/components/ApiReferenceBase.vue

```diff
@@ -161,49 +165,51 @@ if (props.configuration?.authentication) {
       @openSwaggerEditor="gettingStartedModal.hide()"
       @updateContent="handleCloseModal(() => setRawSpecRef($event))" />
   </FlowModal>
-  <ResetStyles v-slot="{ styles }">
-    <ApiReferenceLayout
-      v-bind="$attrs"
-      :class="styles"
-      :configuration="currentConfiguration"
-      :parsedSpec="parsedSpecRef"
-      :rawSpec="rawSpecRef"
-      :swaggerEditorRef="swaggerEditorRef"
-      @changeTheme="$emit('changeTheme', $event)"
-      @updateContent="(newContent: string) => setRawSpecRef(newContent)">
-      <!-- Passes up all the slots except `editor` with typed slot props -->
-      <template
-        v-for="(_, name) in slots"
-        #[name]="scope">
-        <slot
-          :name="name"
-          v-bind="scope" />
-      </template>
-      <template
-        v-if="LazyLoadedSwaggerEditor"
-        #editor>
-        <LazyLoadedSwaggerEditor
-          ref="swaggerEditorRef"
-          :error="errorRef"
-          :proxyUrl="currentConfiguration.proxy"
-          :theme="currentConfiguration.theme"
-          :value="rawSpecRef"
-          @changeTheme="$emit('changeTheme', $event)"
-          @contentUpdate="(newContent: string) => setRawSpecRef(newContent)">
-          <template #tab-items>
-            <HeaderTabButton
-              v-if="isMobile"
-              @click="handleGettingStarted">
-              Getting Started
-            </HeaderTabButton>
-          </template>
-          <template #editor-input="editorInputProps">
-            <slot
-              name="editor-input"
-              v-bind="editorInputProps" />
-          </template>
-        </LazyLoadedSwaggerEditor>
-      </template>
-    </ApiReferenceLayout>
+  <ResetStyles v-slot="{ styles: reset }">
+    <ScrollbarStyles v-slot="{ styles: scrollbars }">
```

Comment:
This approach is not really making sense to me. We have an existing `scrollbar.css` in the closed source repo. `ApiReference` base is what we consume there. Should be not be applying these styles in the `ApiReference` component?

Reply:
Followup question is should ThemeStyles be scoped the way we're scoping the reset